packages/toolpad-components/src/DatePicker.tsx

```diff
@@ -14,6 +14,7 @@ export interface Props extends DesktopDatePickerProps<string, Dayjs> {
   variant: 'outlined' | 'filled' | 'standard';
   size: 'small' | 'medium';
   sx: any;
+  defaultValue: string;
```

Comment:
Not from this PR but shouldn't this props object be called `DatePickerProps` instead of `Props`? Especially as it's being exported.

Reply:
Done